CactbotOverlay/CactbotEventSource.cs

```diff
@@ -804,11 +834,14 @@ private class NotifyState {
       public FFXIVProcess.WarriorJobData war = new FFXIVProcess.WarriorJobData();
       public FFXIVProcess.DarkKnightJobData drk = new FFXIVProcess.DarkKnightJobData();
       public FFXIVProcess.PaladinJobData pld = new FFXIVProcess.PaladinJobData();
+      public FFXIVProcess.GunbreakerJobData gnb = new FFXIVProcess.GunbreakerJobData();
       public FFXIVProcess.BardJobData brd = new FFXIVProcess.BardJobData();
+      public FFXIVProcess.DancerJobData dnc = new FFXIVProcess.DancerJobData();
       public FFXIVProcess.NinjaJobData nin = new FFXIVProcess.NinjaJobData();
       public FFXIVProcess.BlackMageJobData blm = new FFXIVProcess.BlackMageJobData();
       public FFXIVProcess.WhiteMageJobData whm = new FFXIVProcess.WhiteMageJobData();
-      public FFXIVProcess.SummonerAndScholarJobData smn_sch = new FFXIVProcess.SummonerAndScholarJobData();
+      public FFXIVProcess.SummonerJobData smn = new FFXIVProcess.SummonerJobData();
+      public FFXIVProcess.ScholarJobData sch = new FFXIVProcess.ScholarJobData();
```

Comment:
I've added new handlers for gunbreaker & dancer, and due to how different the classes are now in Shadowbringers I've split summoner and scholar into their own seperate handlers.